### ISSUE TITLE
How to extract feature importance in PC1 of PCoA?

### ISSUE THREAD

**Issue by wanglu2014** [joey711/phyloseq#1511]
We can extract feature projection as feature importance in PC1 axis, such as the "components_" attribute in https://scikit-learn.org/stable/modules/generated/sklearn.decomposition.PCA.html

How can we also extract feature importance from PCoA in phyloseq/vegan?

The feature importance is the contribution of each feature to the PC1, not the coordinate of each sample.
An example with displayed output contributions result:
http://www.sthda.com/english/articles/31-principal-component-methods-in-r-practical-guide/118-principal-component-analysis-in-r-prcomp-vs-princomp/#access-to-the-pca-results
